src/assemblyinfo/assemblyinfo.FSharp.Compiler.Private.dll.fs

```diff
@@ -71,4 +71,10 @@ open System.Runtime.InteropServices
 [<assembly: System.Reflection.AssemblyFileVersion("2017.06.27.0")>]
 #endif
 
+#if HAVE_VS_SDK
```

Comment:
The exact same is in place for FSharp.Compiler.Server.Shared, so this isn't unique in that respect.